app/views/courts/_office.html.erb

```diff
@@ -45,28 +45,28 @@
     <table class="table table-condensed table-bordered table-striped">
       <thead>
         <th colspan="2">
-          Otváracie hodiny <%= icon_tag :time %>
+          <%= t '.opening_hours' %> <%= icon_tag :time %>
         </th>
       </thead>
       <tbody>
         <tr>
-          <td class="span1">Pondelok:</td>
+          <td class="span1"><%= t '.monday' %>:</td>
```

Comment:
Does this work? In translations, you have `office.monday` path. On the other hand, look at `t(:'date.day_names')` - might be handy to use existing translations, day names does not change, I think.